cmdline/configfile.md

```diff
@@ -1,78 +1,126 @@
-command-line options, making them rather hard to work with. Sometimes the
-length of the command line you want to enter even hits the maximum length your
-command-line system allows. The Microsoft Windows command prompt being an
-example of something that has a fairly small maximum line length.
+Curl commands with multiple command-line options can become cumbersome to work with. The number of characters can even exceed the maximum length allowed by your terminal application. 
 
-To aid such situations, curl offers a feature we call "config file". It allows
-you to write command-line options in a text file instead and then tell curl to
-read options from that file in addition to the command line.
+To aid such situations, curl allows you to write command-line options in a plain text config file and tell curl to read options from that file when applicable. 
 
-You tell curl to read more command-line options from a specific file with the
--K/--config option, like this:
+You can also use config files to assign data to variables and transform the data with functions, making them incredibly useful. This is discussed in the ["Variables"](https://everything.curl.dev/cmdline/variables) section.
 
-    curl -K cmdline.txt http://example.com
+Some examples below contain multiple lines for readability. The forward slash (`\`) is used to instruct the terminal to ignore the newline.
 
-…and in the `cmdline.txt` file (which, of course, can use any filename you
-please) you enter each command line per line:
+## Options
 
-    # this is a comment, we ask to follow redirects
-    --location
-    # ask to do a HEAD request
-    --head
+Using the `-K` or long form `--config` option tells curl to read from a config file.
 
-The config file accepts both short and long options, exactly as you would
-write them on a command line. As a special extra feature, it also allows you
-to write the long format of the options without the leading two dashes to make
-it easier to read. Using that style, the config file shown above can
-alternatively be written as:
+## Example 1
 
-    # this is a comment, we ask to follow redirects
-    location
-    # ask to do a HEAD request
-    head
+```
+curl  \
+    --config configFile.txt \
+    --url https://example.com
+```
 
-Command line options that take an argument must have its argument provided on
-the same line as the option. For example changing the User-Agent HTTP header
-can be done with
+The file path specified is relative to the current directory in your terminal.
 
-    user-agent "Everything-is-an-agent"
+You can name the config file whatever you'd like. `configFile.txt` is used for simplicity in the example above.
 
-To allow the config files to look even more like a true config file, it also
-allows you to use '=' or ':' between the option and its argument. As you see
-above it is not necessary, but some like the clarity it offers. Setting the
+## Syntax
+
+Enter one command per line. Use a hashtag for comments:
```

Comment:
The name of the symbol is not "hashtag" is it? hash, number sign and pound sign seem to be the most commonly used names. A "hashtag" is when the hash is used combined with a following name in various scenarios.

Reply:
Yeah, I see how using "hash symbol" vs "hashtag" would be better since "hashtag" is more commonly used to reference keywords like you said. This has been updated.